Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -267,66 +257,12 @@ function (array $types, NodeType $superType) use ($self) {
         return array_unique($types);
     }
 
-
-    /**
-     * Fetches possible usages of the asset and registers nodes that use the asset as changed.
-     *
-     * @throws NodeTypeNotFound
-     */
-    public function registerAssetChange(AssetInterface $asset): void
```

Comment:
Looking at the tests, asset handling still works, but… why? 😳

Reply:
Because it has ben moved here:
https://github.com/neos/neos-development-collection/pull/5221/files#diff-d8a3ff4c9ee4f131316bbcbeff82bd703668086d19a94885d610d65e51fdbbd2R33